packages/fxa-auth-server/lib/senders/emails/templates/verificationReminderSecond/index.mjml

```diff
@@ -2,4 +2,24 @@
   # License, v. 2.0. If a copy of the MPL was not distributed with this
   # file, You can obtain one at http://mozilla.org/MPL/2.0/. %>
 
-<%- include('/partials/verificationReminder/index.mjml') %>
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-header">
+      <span data-l10n-id="verificationReminderSecond-title">Still there?</span>
+    </mj-text>
+  </mj-column>
+</mj-section>
+
+<mj-section>
+  <mj-column>
+    <mj-text css-class="text-sub-body">
+      <span data-l10n-id="verificationReminderSecond-description">Almost a week ago you created a { -product-firefox-account } but never verified it. We’re worried about you.</span>
```

Comment:
Since this is the English fallback text, we shouldn't use placeholders here, this should be "created a Firefox account...'

Reply:
Ah, make sense. Thanks for pointing this out.